scripts/insolard/launch.sh

```diff
@@ -0,0 +1,118 @@
+set -e
+
+GINSIDER_DIR=logicrunner/goplugin/ginsider-cli
+GINSIDER_BIN=$GINSIDER_DIR/ginsider-cli
+TEST_DATA=testdata
+INSOLARD=$TEST_DATA/functional/insolard
```

Comment:
May be its should be separate special directory? Not connected to test in naming